docker-compose.yml

```diff
@@ -44,7 +29,7 @@ services:
     # mdillon/postgis is postgres + PostGIS (only). if you need additional
     # extensions, it's probably time to create a purpose-built image with all
     # necessary extensions. sorry, and thanks for adding support for them!
-    image: "mdillon/postgis:9.6"
+    image: "mdillon/postgis:14.5"
```

Comment:
I'm not sure about changing the version of this postgres versus the postgres-12 that's also here